src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ContentDialog/UnoSamples_Tests.ContentDialog.cs

```diff
@@ -417,15 +416,15 @@ public void ContentDialog_Simple_NotLightDismissible()
 			{
 				if (AppInitializer.GetLocalPlatform() == Platform.Android)
 				{
-					// the status bar area needs to be excluded for image comparison
+					// the status bar area should be excluded for image comparison
+					// 24dp is default height of status bar
```

Comment:
I would imagine that it depends of the scaling of the emulator. We don't  a way to get the actual value instead of hard coding it ?